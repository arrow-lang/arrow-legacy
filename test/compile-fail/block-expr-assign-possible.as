let a;

if false {
} else if { a = 20; a > 50; } {
}

let b = a; // error
