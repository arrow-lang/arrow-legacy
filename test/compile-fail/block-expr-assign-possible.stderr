[0;37mtest/compile-fail/block-expr-assign-possible.as:7.9-10: [0;31merror: [1;37muse of possibly uninitialized variable 'a'[0m
